scripts/entity/components/movement_components/base_movement_component.gd

```diff
@@ -40,27 +37,12 @@ func _ready() -> void:
 ## Determines whether or not the entity is grounded or airborne this frame and
 ## saves the current state for use next frame.
 func _physics_process(_delta: float) -> void:
-	emit_movement_signals()
+	_emit_movement_signals()
 	_try_to_run_grounded_events()
 	_was_grounded = is_on_floor()
 	_previous_horizontal_velocity = velocity.x
```

Comment:
Thought for later: If we keep having "do at end of processing" statements like this, it might be good to group into a function. Probably not needed for now.